Formula/capnp.rb

```diff
@@ -39,6 +39,11 @@ def install
   end
 
   test do
-    system "#{bin}/capnp", "--version"
+    file = testpath/"test.capnp"
+    text = "\"Is a happy little duck\""
+
+    file.write Utils.popen_read("#{bin}/capnp id").chomp + ";\n"
+    file.append_lines "const dave :Text = #{text};"
+    assert_match text, shell_output("#{bin}/capnp eval #{file} dave")
```

Comment:
what if it's printing "Is a happy little duck" is invalid syntax... won't that match?